Anchorage.swift

```diff
@@ -262,214 +264,221 @@ infix operator ~: PriorityPrecedence
         return EdgeAnchors(top: topLayoutGuide.bottomAnchor, leading: view.leadingAnchor, bottom: bottomLayoutGuide.topAnchor, trailing: view.trailingAnchor)
     }
-
+    
 }
 
 extension UILayoutGuide: EdgeAnchorsProvider {
-
+    
     public var edgeAnchors: EdgeAnchors {
         return EdgeAnchors(top: topAnchor, leading: leadingAnchor, bottom: bottomAnchor, trailing: trailingAnchor)
     }
-
+    
 }
 
 // MARK: - LayoutEdge
 
-private enum LayoutEdge {
-
+enum LayoutEdge {
+    
     case top, leading, bottom, trailing
     static let Horizontal = [leading, trailing]
     static let Vertical = [top, bottom]
     static let All = [top, leading, bottom, trailing]
-
-    var axis: UILayoutConstraintAxis {
+    
+    var axis: ConstraintAxis {
         switch self {
         case .top, .bottom:
             return .vertical
         case.leading, .trailing:
             return .horizontal
         }
     }
-
-    func transform(constant: CGFloat) -> CGFloat {
+    
+    func transform(constant theConstant: CGFloat) -> CGFloat {
         switch self {
         case .top, .leading:
-            return constant
+            return theConstant
         case .bottom, .trailing:
-            return -constant
+            return -theConstant
         }
     }
+    
+}
 
+enum ConstraintAxis: Int {
+    case vertical
+    case horizontal
 }
 
 // MARK: - EdgeAnchors
 
 public struct EdgeAnchors: LayoutAnchorType {
-
+    
     private var top: NSLayoutYAxisAnchor
     private var leading: NSLayoutXAxisAnchor
     private var bottom: NSLayoutYAxisAnchor
     private var trailing: NSLayoutXAxisAnchor
-
+    
     private var includedEdges = LayoutEdge.All
-
-    fileprivate init(top: NSLayoutYAxisAnchor, leading: NSLayoutXAxisAnchor, bottom: NSLayoutYAxisAnchor, trailing: NSLayoutXAxisAnchor) {
+    
+    init(top: NSLayoutYAxisAnchor, leading: NSLayoutXAxisAnchor, bottom: NSLayoutYAxisAnchor, trailing: NSLayoutXAxisAnchor) {
         self.top = top
         self.leading = leading
         self.bottom = bottom
         self.trailing = trailing
     }
-
-    fileprivate func filter(_ filter: [LayoutEdge]) -> EdgeAnchors {
+    
+    func filter(forEdges edges: [LayoutEdge]) -> EdgeAnchors {
         var filteredAnchors = self
-        filteredAnchors.includedEdges = includedEdges.filter { filter.contains($0) }
-
+        filteredAnchors.includedEdges = includedEdges.filter { edges.contains($0) }
+        
         return filteredAnchors
     }
-
-    public func activeConstraintsEqualToEdges(_ anchor: EdgeAnchors?, constant c: CGFloat = 0.0, priority: UILayoutPriority = UILayoutPriorityRequired) -> EdgeConstraints {
-        return constraintsForAnchors(anchor, constant: c, priority: priority, builder: ConstraintBuilder(horizontal: ==, vertical: ==))
+    
+    public func activeConstraints(equalToEdges anchor: EdgeAnchors?, constant c: CGFloat = 0.0, priority: UILayoutPriority = UILayoutPriorityRequired) -> EdgeConstraints {
```

Comment:
Swift 3.0 naming conventions should drop the last `Edges` so -- `equalTo` not `equalToEdges`